templates/jlink/README.md

```diff
@@ -18,26 +20,31 @@ prepare UBI9 OpenJDK ImageStreams with `jlink-dev` support.
 
         cekit --descriptor ubi9-openjdk-17.yaml build docker
 
-2. Within your OpenShift project,
+2. Create an OpenShift project and namespace
+
+        oc new-project jlink-dev
+        oc create namespace jlink
```

Comment:
You only need one of these commands. I suggest keepign the `oc new-project jlink-dev` as this is the standard workflow.